lib/OpenQA/Task/Bug/Limit.pm

```diff
@@ -0,0 +1,84 @@
+# Copyright (C) 2018-2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package OpenQA::Task::Bug::Limit;
+use Mojo::Base 'Mojolicious::Plugin';
+
+use File::Spec::Functions 'catfile';
+use File::Basename qw(basename dirname);
+use OpenQA::Utils qw(:DEFAULT imagesdir);
+use Data::Dump 'pp';
+use Try::Tiny;
+
+sub register {
+    my ($self, $app) = @_;
+    $app->minion->add_task(limit_bugs => sub { _limit($app, @_) });
```

Comment:
This indirection is pointless. Use:
```perl
$app->minion->add_task(limit_bugs => \&_limit);
```
And:
```perl
sub _limit {
    my $job = shift;

    my $app = $job->app;
    ...
```

Reply:
Then it is pointless in `lib/OpenQA/Task/Asset/Limit.pm` where I copied it from as well.

Reply:
Correct.